ruby/src/api.cc

```diff
@@ -136,11 +136,7 @@ namespace leatherman { namespace ruby {
         } else {
             LOG_INFO("ruby was already loaded.");
         }
-        try {
-            return unique_ptr<api>(new api(move(library)));
-        } catch (lth_lib::missing_import_exception& ex) {
-            throw ex;
-        }
+        return unique_ptr<api>(new api(move(library)));
```

Comment:
Ah, move operator is also deleted. I guess forgot it then, we can use a unique_ptr internally.